app/assets/stylesheets/courses.scss

```diff
@@ -0,0 +1,14 @@
+.date .collection-item {
+  padding-left: 24px !important;
+  padding-right: 24px !important;
+}
+
+.date .collection-item p {
+  line-height: 1rem;
+}
+
+.date p {
+  margin: 0;
+  font-size: 0.85rem;
+  color: #676464;

```

Comment:
Use a variable instead of hex

Reply:
Change this for assessments.scss as well

Also you are copying over css, consider making a separate css file for these styles shared by both courses and assessments